bitnami/wordpress/templates/deployment.yaml

```diff
@@ -147,6 +151,8 @@ spec:
               value: {{ .Values.wordpressBlogName | quote }}
             - name: WORDPRESS_SKIP_INSTALL
               value: {{ ternary "yes" "no" .Values.wordpressSkipInstall | quote }}
+            - name: WORDPRESS_READ_ONLY_CONFIG
+              value: {{ ternary "yes" "no" (or (not (empty .Values.wordpressConfiguration)) (not (empty .Values.existingWordPressConfigurationSecret))) | quote }}
```

Comment:
I see that the boolean expression below (the one that adds the volumeMount) doesn't require the `not empty`. Is it necessary or not?

Reply:
Here it's necessary since "ternary" only admits a boolean for its third parameter and it complained without the `not empty` while it did not complain in the "if"